app/src/main/res/layout/fragment_messages_master_list.xml

```diff
@@ -1,23 +1,46 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
 
     <data>
 
         <variable
             name="viewModel"
-            type="com.teamagam.gimelgimel.app.message.viewModel.MessagesViewModel"/>
+            type="com.teamagam.gimelgimel.app.message.viewModel.MessagesViewModel" />
     </data>
 
-    <android.support.v7.widget.RecyclerView
-        android:id="@+id/fragment_messages_recycler"
-        android:name="com.teamagam.gimelgimel.app.view.fragments.MessagesMasterFragment"
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        xmlns:tools="http://schemas.android.com/tools"
+    <FrameLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:choiceMode="singleChoice"
-        app:layoutManager="LinearLayoutManager"
-        tools:context=".app.message.view.MessagesFragment"
-        tools:listitem="@layout/recycler_message_list_item_other"/>
+        android:background="#E5DDD5">
```

Comment:
extract color string to values/colors.
It's easy, move cursor to it and alt+return